replaying-share/src/main/java/com/jakewharton/rx/ReplayingShare.java

```diff
@@ -43,32 +47,58 @@
     return (ReplayingShare<T>) INSTANCE;
   }
 
+  /**
+   * Creates a `ReplayingShare` transformer with a default value which will be emitted downstream
+   * on subscription if there is not any cached value yet.
+   */
+  public static <T> ReplayingShare<T> createWithDefault(T defaultValue) {
+    return new ReplayingShare<>(defaultValue);
+  }
+
   private ReplayingShare() {
```

Comment:
Let's delete this constructor and move the non-null check into `createWithDefault`.